build failed

<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx:147:11
Type error: Type 'MunicipalityData[]' is not assignable to type 'Municipality[]'.
  Property 'code' is missing in type 'MunicipalityData' but required in type 'Municipality'.

  145 |           title="Renovatie (Gebouwen)"
  146 |           data={data}
> 147 |           municipalities={municipalities}
      |           ^
  148 |           metric="ren"
  149 |           label="Aantal"
  150 |           slug="vergunningen-goedkeuringen"
Error: Process completed with exit code 1.
@claude

--- a/embuild-analyses/src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx
+++ b/embuild-analyses/src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx
@@ -16,7 +16,7 @@ type DataRow = {
 }
 
 type MunicipalityData = {
-  m: number
+  code: number
   name: string
 }
 

